lib/galaxy/datatypes/hubAssembly.py

```diff
@@ -0,0 +1,59 @@
+"""
+HubAssembly datatype
+"""
+import logging
+import galaxy.version as version
+
+# Support for Galaxy <= 16.01
+if version.VERSION_MAJOR <= "16.01":
+    from galaxy.datatypes.images import Html
+else:
+    from galaxy.datatypes.text import Html
```

Comment:
I don't get why theses lines prevent the merging in Galaxy codebase. Does it harm something? Just to know so I won't do the same errors in the future.